<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit2</class>
 <widget class="QWidget" name="edit2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>788</width>
    <height>539</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理数据</string>
  </property>
  <widget class="QLabel" name="change_lb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入一组数据</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_op">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <item>
    <property name="text">
     <string>查看</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>增加</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>修改</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>删除</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_db">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>31</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>市民基本信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>各区风险等级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>各街道管控类型</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>市民核酸检测报告</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>市民疫苗报告</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>核酸检测点</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>疫苗定点医院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>核酸检测业务类型</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>疫苗类型</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择需要管理的库</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="add_tb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择操作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请双击选择需要修改的单元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择需要删除的行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
